### DISCUSSION TITLE
Change Jetton token template

### DISCUSSION THREAD

**Discussion by vodanh86** [ton-org/game-engines-sdk#11] [General]
Dear Barinbritva
I am using example from this repo: https://github.com/ton-community/flappy-bird. I want to change the Jetton symbol and image, so I update file content-templates.ts in scripts/tokens/flap.
After I run setup.sh. I have the new jetton token but it does not have the  new symbol and image. 
![Screenshot 2024-07-24 at 10 39 44](https://github.com/user-attachments/assets/b0df6fe9-18d9-434c-a36f-d42f9c35ea23)
Can you please help me how to customize the jetton.

**Comment by barinbritva** [comment]
@vodanh86 hi,

It seems, you run `setup.sh` with default values in `scripts\tokens\flap\content-template.ts`, right?
If so, you need to remove or comment `JETTON_ADDRESS` key and run `setup.sh` script again. Don't forget to backup current `.env` file to not lost other settings

You also can just run `scripts\deploy-jetton.ts` with ts-node and put new jetton address to `.env` manually

**Reply by vodanh86** [reply]
Dear Barinbritva,
I can change the JETTON token. 
Thank you so much.

**Reply by barinbritva** [reply]
I'm glad to hear that!
